translation-web/src/main/resources/translation.properties

```diff
@@ -16,4 +16,9 @@ auth.write.enabled=true
 translation.google.usehttpclient=false
 
 # For using dummy services
-#use.dummy.services=true
\ No newline at end of file
+#use.dummy.services=true
+
+spring.redis.database=0
```

Comment:
we should have only one property, redis.connection.url

swithc to use Lettuce instead of Jedis https://docs.spring.io/spring-data/data-redis/docs/current/reference/html/#redis:connectors:lettuce

and use the following method to parse the URL to a RedisConfigurationObject

https://docs.spring.io/spring-data/redis/docs/current/api/org/springframework/data/redis/connection/lettuce/LettuceConnectionFactory.html#createRedisConfiguration(java.lang.String)

Reply:
also, the connectionURL is sensitive information, it is to be documented in the translation.user.properties.template and removed from this file